vp/devel/hpc.repl

```diff
@@ -4,4 +4,4 @@ cpu_hpc: CPU.RiscV64 @ sysbus
     timeProvider: empty
 
 sdram: Memory.MappedMemory @ sysbus 0x10000000
-    size: 0x70000000
+    size: 0x60000000
```

Comment:
Onko niin että meillä on tämä muistin koko generoitunut väärin? Mikä on syy sille että tämän kokoa muutetaan. Asia on syytä selvittää perinpohjaisesti :)

Reply:
Joo näin on. Pidän todennäköisimpänä että olen itse tehnyt tuohon jonkun virhekorjauksen. Tämä on oikein miten laitoit.